agent/api/ecsclient/client_test.go

```diff
@@ -84,9 +84,10 @@ func NewMockClient(ctrl *gomock.Controller,
 
 	return NewMockClientWithConfig(ctrl, ec2Metadata, additionalAttributes,
 		&config.Config{
-			Cluster:            configuredCluster,
-			AWSRegion:          "us-east-1",
-			InstanceAttributes: additionalAttributes,
+			Cluster:                 configuredCluster,
+			AWSRegion:               "us-east-1",
+			InstanceAttributes:      additionalAttributes,
+			IPv6PortBindingExcluded: config.BooleanDefaultTrue{Value: config.ExplicitlyEnabled},
```

Comment:
nit - maybe a more appropriate name would be `ShouldExcludeIPv6PortBinding`? Technically at the moment that this variable is checked it's not excluded yet.

Reply:
Update the config variable to `ShouldExcludeIPv6PortBinding` as recommended. Thanks!